<ui version="4.0" >
 <class>TPOscillographDialog</class>
 <widget class="QDialog" name="TPOscillographDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Осциллограф</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QwtPlot" name="qwtpOscillograph" />
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="pbStart" >
       <property name="text" >
        <string>Старт</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbStop" >
       <property name="text" >
        <string>Стоп</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbClearResistor" >
       <property name="text" >
        <string>Сбросить резистор</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbExit" >
       <property name="text" >
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>TPOscillographDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>515</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>281</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbExit</sender>
   <signal>clicked()</signal>
   <receiver>pbStop</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>515</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>132</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
